src/styles/utils/_variables.scss

```
@@ -0,0 +1,12 @@
+$primary-color: hsl(193, 100%, 43%);
+$dark-color: #000;
+$default-color: #fff;
+$high-blue-color: #060b35;
+$grey-color: #616070;
+$border-color: #f3f3f3;
+$card-width: 200px;
+$fixed-height: 60px;
+$card-between-hor: 50px;
+$card-between-ver: 48px;
```

adjust gaps:
```suggestion
$card-between-hor: 48px;
$card-between-ver: 46px;
```
and try the tests again